simpleclient_hibernate/src/main/java/io/prometheus/client/hibernate/HibernateStatisticsCollector.java

```diff
@@ -0,0 +1,282 @@
+            statistics.getUpdateTimestampsCacheHitCount()
+        ),
+        createCounter(
+            "hibernate_update_timestamps_cache_miss_total",
+            "Get the global number of tables for which no update timestamps was not found in cache",
+            statistics.getUpdateTimestampsCacheMissCount()
+        ),
+        createCounter(
+            "hibernate_update_timestamps_cache_put_total",
+            "Get the global number of timestamps put in cache",
+            statistics.getUpdateTimestampsCachePutCount()
+        )
+
+    );
+  }
+
+  private List<MetricFamilySamples> getEntityMetrics() {
+    return Arrays.<MetricFamilySamples>asList(
+
+        createCounter(
+            "hibernate_entity_delete_total",
+            "Get global number of entity deletes",
+            statistics.getEntityDeleteCount()
+        ),
+        createCounter(
+            "hibernate_entity_insert_total",
+            "Get global number of entity inserts",
+            statistics.getEntityInsertCount()
+        ),
+        createCounter(
+            "hibernate_entity_load_total",
+            "Get global number of entity loads",
+            statistics.getEntityLoadCount()
+        ),
+        createCounter(
+            "hibernate_entity_fetch_total",
+            "Get global number of entity fetches",
+            statistics.getEntityFetchCount()
+        ),
+        createCounter(
+            "hibernate_entity_update_total",
+            "Get global number of entity updates",
+            statistics.getEntityUpdateCount()
+        ),
+
+        createCounter(
+            "hibernate_collection_load_total",
+            "Global number of collections loaded",
+            statistics.getCollectionLoadCount()
+        ),
+        createCounter(
+            "hibernate_collection_fetch_total",
+            "Global number of collections fetched",
+            statistics.getCollectionFetchCount()
+        ),
+        createCounter(
+            "hibernate_collection_update_total",
+            "Global number of collections updated",
+            statistics.getCollectionUpdateCount()
+        ),
+        createCounter(
+            "hibernate_collection_remove_total",
+            "Global number of collections removed",
+            statistics.getCollectionRemoveCount()
+        ),
+        createCounter(
+            "hibernate_collection_recreate_total",
+            "Global number of collections recreated",
+            statistics.getCollectionRecreateCount()
+        )
+
+    );
+  }
+
+  private List<MetricFamilySamples> getQueryExecutionMetrics() {
+    return Arrays.asList(
+
+        createCounter(
+            "hibernate_query_execution_total",
+            "Get global number of executed queries",
+            statistics.getQueryExecutionCount()
+        ),
+
+        createGauge(
+            "hibernate_query_execution_seconds_max",
+            "Get the time in milliseconds of the slowest query.",
```

Comment:
AFAIK this is since the session factory startup. I wasn't sure whether it makes sense to add these when I wrote the code, so I've no problem with dropping them.